### DISCUSSION TITLE
Using gnome-shell-wsmatrix with Ubuntu 23.10 and GNOME 45.0

### DISCUSSION THREAD

**Discussion by mtcq** [mzur/gnome-shell-wsmatrix#272] [Q&A] [answered]
Hi everyone,
recently I have updated my Ubuntu from 23.04 to 23.10, and since then I'm not being able to use wsmatrix.
I have installed wsmatrix following the "Manual Linux Method", as I do after every Ubuntu update since I've started using wsmatrix. However, this time, I still didn't manage to make it work properly.
When I open wsmatrix via "gnome-extensions prefs wsmatrix@martin.zurowietz.de", I see the text :
"Some features of the extention are disabled until GNOME Shell 45.1."
So, I believe that everything might work if I update my GNOME shell to 45.1.

Does anyone have any experience with this?
Also, in case I decide to update my GNOME Shell from 45.0 to 45.1 using Ubuntu 23.10, does anyone have a suggestion or recommendation on how to do that?

Thank you very much!

**Comment by mzur** [comment, marked as answer]
Hi, the extension should work but some features are disabled or work in an unexpected way in GNOME Shell 45.0. Since you can see the new preferences window, the extension installation has worked. Do you also the the 2D workspaces and the workspaces popup?

The fixes that are required for the extension to fully work again are already merged into GNOME Shell but not released yet. I expect them to arrive in GNOME SHell 45.1 which also isn't released, yet.

**Reply by mtcq** [reply]
Dear mzur,
thank you very much for the quick and kind response!
>Do you also the the 2D workspaces and the workspaces popup?
  

No... Unfortunately, I cannot use a 2D workspace. Even with wsmatrix installed, my workspaces are still in a horizontal line, and I don't manage to make a matrix workspace (even if I press the super key). So, unfortunately, the main feature (to have a 2d grid) doesn't seem to work for me.

**Reply by mzur** [reply]
Do you have the "Extensions" app installed? It will show if there are errors when the extension should be enabled.

**Reply by mtcq** [reply]
I believe I have the Extensions app installed, but it doesn't provide any particular error message. I took a screenshot which illustrates the problem I have. Where I set a 4 by 4 four worskpace grid, but I have 16 workspaces in a horizontal line.

![Capture d’écran du 2023-11-22 11-26-18](https://github.com/mzur/gnome-shell-wsmatrix/assets/42988902/5cbe275c-5ee5-452f-b684-4cccef1f5790)

**Reply by mtcq** [reply]
I've just noticed that the wsmatrix is now working! I don't know exactly what changed here. When I press the power key, I still see my workspaces in a line. However, when I change workspaces by holding ctrl+alt, I do have access to a two-dimensional grid!!

Thank you very much for the support!
